docs/_data/options.js

```diff
@@ -1,178 +1,201 @@
-		default: 'en',
-		selectField: true,
-		text: 'en',
-	},
-	{
-		name: ' --hostname',
-		id: 'hostname',
-		param: '--hostname',
-		desc: 'Set hostname',
-		selected: false,
-		text: '',
-		textField: true,
+		flag: 'port',
+		label: 'Port',
+		description: 'Change the port Hestia uses.',
+		type: 'text',
+		default: '8083',
 	},
 	{
-		name: ' --email',
-		id: 'email',
-		param: '--email',
-		desc: 'Set admin email',
-		selected: false,
-		text: '',
-		textField: true,
+		flag: 'lang',
+		label: 'Language',
+		description: 'Change the ISO 639-1 language code.',
+		type: 'select',
+		default: 'en',
+		options: [
+			{ label: 'Albanian', value: 'sq' },
+			{ label: 'Arabic', value: 'ar' },
+			{ label: 'Armenian', value: 'hy' },
+			{ label: 'Azerbaijani', value: 'az' },
+			{ label: 'Bengali', value: 'bn' },
+			{ label: 'Bosnian', value: 'bs' },
+			{ label: 'Bulgarian', value: 'bg' },
+			{ label: 'Catalan', value: 'ca' },
+			{ label: 'Croatian', value: 'hr' },
+			{ label: 'Czech', value: 'cs' },
+			{ label: 'Danish', value: 'da' },
+			{ label: 'Dutch', value: 'nl' },
+			{ label: 'English', value: 'en' },
+			{ label: 'Finnish', value: 'fi' },
+			{ label: 'French', value: 'fr' },
+			{ label: 'Georgian', value: 'ka' },
+			{ label: 'German', value: 'de' },
+			{ label: 'Greek', value: 'el' },
+			{ label: 'Hungarian', value: 'hu' },
+			{ label: 'Indonesian', value: 'id' },
+			{ label: 'Italian', value: 'it' },
+			{ label: 'Japanese', value: 'ja' },
+			{ label: 'Korean', value: 'ko' },
+			{ label: 'Kurdish Sorani', value: 'ku' },
+			{ label: 'Norwegian', value: 'no' },
+			{ label: 'Persian', value: 'fa' },
+			{ label: 'Polish', value: 'pl' },
+			{ label: 'Portuguese', value: 'pt' },
+			{ label: 'Portuguese (Brasil)', value: 'pt-br' },
+			{ label: 'Romanian', value: 'ro' },
+			{ label: 'Russian', value: 'ru' },
+			{ label: 'Serbian', value: 'sr' },
+			{ label: 'Simplified Chinese (China)', value: 'zh-cn' },
+			{ label: 'Slovak', value: 'sk' },
+			{ label: 'Spanish', value: 'es' },
+			{ label: 'Swedish', value: 'sv' },
+			{ label: 'Thai', value: 'th' },
+			{ label: 'Traditional Chinese (Taiwan)', value: 'zh-tw' },
+			{ label: 'Turkish', value: 'tr' },
+			{ label: 'Ukrainian', value: 'uk' },
+			{ label: 'Urdu', value: 'ur' },
+			{ label: 'Vietnamese', value: 'vi' },
+		],
+	},
+	{
+		flag: 'hostname',
+		label: 'Hostname',
+		description: 'Set a custom hostname.',
+		type: 'text',
+		default: '',
+	},
+	{
+		flag: 'email',
+		label: 'Email',
+		description: 'Set the admin account email.',
+		type: 'text',
+		default: '',
+	},
+	{
+		flag: 'password',
+		label: 'Password',
+		description: 'Set the admin account password.',
+		type: 'text',
+		default: '',
+	},
+	{
+		flag: 'apache',
+		label: 'Apache',
+		description: 'Web server with htaccess support.',
+		default: 'yes',
+	},
+	{
+		flag: 'phpfpm',
+		label: 'PHP-FPM',
+		description: 'FastCGI Process Manager for executing PHP scripts.',
+		default: 'yes',
```

Comment:
I am sure the default option is no

https://github.com/hestiacp/hestiacp/blob/4153e6c0ff05057610ed6d62fc788f2ef664eb0f/install/hst-install-ubuntu.sh#L58C18-L58C18

Reply:
Looks like this is wrong on the correct install command generator. Fixed in 9e0e9caae.